src/main/java/de/uhd/ifi/se/decision/management/jira/eventlistener/implementation/ConsistencyCheckEventListenerSingleton.java

```diff
@@ -0,0 +1,71 @@
+package de.uhd.ifi.se.decision.management.jira.eventlistener.implementation;
+
+import com.atlassian.jira.event.issue.IssueEvent;
+import de.uhd.ifi.se.decision.management.jira.consistency.ConsistencyCheckEventTrigger;
+import de.uhd.ifi.se.decision.management.jira.consistency.implementation.StatusClosedTrigger;
+import de.uhd.ifi.se.decision.management.jira.consistency.implementation.WorkflowDoneTrigger;
+import de.uhd.ifi.se.decision.management.jira.eventlistener.IssueEventListener;
+import de.uhd.ifi.se.decision.management.jira.eventlistener.Observabel;
+import de.uhd.ifi.se.decision.management.jira.eventlistener.Subscriber;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collector;
+
+public class ConsistencyCheckEventListenerSingleton implements IssueEventListener, Observabel {
+
+	List<ConsistencyCheckEventTrigger> consistencyCheckEventTriggerList;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5586048)